### DISCUSSION TITLE
Retina works fine with trealla

### DISCUSSION THREAD

**Discussion by josd** [trealla-prolog/trealla-prolog#514] [Show and tell]
https://github.com/KNowledgeOnWebScale/retina is daily tested with trealla and it is so nice to see
![image](https://github.com/trealla-prolog/trealla/assets/193444/ea20680c-9d79-456c-a0f4-7a109fc33f57)

**Comment by infradig** [comment]
I test it regularly as well, it's picked up some useful bugs.

**Comment by josd** [comment]
There is now also a paper [RDF Surfaces: Enabling Classical Negation on the Semantic Web](
https://arxiv.org/abs/2406.10659) mentioning Scryer and Trealla
> Retina [15] is also implemented in Prolog and can be run in Trealla [12] and Scryer [13]. The code base is a rewrite  of EYE targeted to processing RDF Surfaces.
